src/components/AdminProfile/SendLinkModal/SendLinkModal.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  ChakraProvider,
+  Text,
+  Input,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+} from '@chakra-ui/react';
+import Checkmark from '../../../assets/Check.png';
+import styles from './SendLinkModal.module.css';
+
+const ModalOne = ({ count, setCount }) => {
+  return (
+    <div className={styles['modal-content']}>
+      <Text fontSize="xl" fontWeight="bold" style={{ alignSelf: 'flex-start' }}>
+        Send registration link via email:
+      </Text>
+      <Input
+        placeholder="ex: jdoeFYA@gmail.com"
+        size="lg"
+        color="#7D7D7D"
+        bg="#F6F6F6"
+        className={styles['modal-one-input']}
+      />
+      <Button
+        onClick={() => setCount(count + 1)}
+        color="white"
+        bg="#345E80"
+        iconSpacing="120px"
+        className={styles['modal-one-button']}
+      >
+        Send Link
+      </Button>
+    </div>
+  );
```

Comment:
I think we should error checking here for the email, since we want to make sure users are sending registration links to email addresses and not just any random string of text, we can ask design to implement what the error should look like

Reply:
I'll make a note to add error checking for email in the auth PR! Marking as resolved here